Revvvolution/Container-Flow
# Database Table Creation and Seed Data Files Added

## Description
Created two executable .sql files - one for creation of the database and tables, and the other for inserting seed data into the created database tables.

## Type of changes

- [x] Database initialization files
- [x] File cleanup
- [x] Launch settings adjusted


## Testing Instructions
1. If you haven't created the database, please do so:
    - In your Visual Studio solution, open the SQL Server Object Explorer View and open a new SQL query.
    - ### Please open these files or paste to the query and execute the following code (You may need to connect your query to your current database): 
   - [SQL Database / Table Creation Data](https://github.com/Revvvolution/Container-Flow/blob/rev-sql-dbquery/SQL/01_ContainerFlow_Create_DB.sql)
   - [SQL Seed Data for Project](https://github.com/Revvvolution/Container-Flow/blob/rev-sql-dbquery/SQL/02_ContainerFlow_Seed_Data.sql)
2. Check and verify that the database items are displayed properly executing the following query selectors:
    - `SELECT * FROM Item`
    - `SELECT c.Id AS 'Container Id', c.Name AS 'Container Name', i.Name AS 'Item Name' FROM Container c LEFT JOIN ContainerItem ci ON ci.ContainerId = c.Id LEFT JOIN Item i ON i.Id = ci.ItemId WHERE c.Id = 1;`


## Checklist:
